pkg/providers/vsphere/vsphere.go

```diff
@@ -339,8 +366,10 @@ func (p *vsphereProvider) SetupAndValidateCreateCluster(ctx context.Context, clu
 		logger.Info("Skipping check for whether control plane ip is in use")
 	}
 
-	if err := p.validator.validateVsphereUserPrivs(ctx, vSphereClusterSpec); err != nil {
-		return fmt.Errorf("validating vsphere user privileges: %v", err)
+	if !p.skippedValidations[validations.VSphereUserPriv] {
+		if err := p.validator.validateVsphereUserPrivs(ctx, vSphereClusterSpec); err != nil {
+			return fmt.Errorf("validating vsphere user privileges: %v", err)
```

Comment:
(same as below) Do we have an argument for not running a test case that triggers this error? It feels like the test case of verifying the error can be thrown would be an important one. Open to hearing counter arguments, those are just my first thoughts.